CONTRIBUTING.md

```diff
@@ -23,7 +23,9 @@ When reporting a bug in the navigation SDK itself, please indicate:
 
 ## Setting up a development environment
 
-To contribute code changes to this project, use either Carthage or Swift Package Manager to set up a development environment. Carthage and the Xcode project in this repository are important for making sure dependent projects can use Carthage and for checking your work in the included Example or Example-CarPlay application. Swift Package Manager only builds MapboxCoreNavigation.
+To contribute code changes to this project, use Swift Package Manager to set up a development environment. The repository includes:
+1. `MapboxNavigation-SPM.xcodeproj` with Example or Example-CarPlay applications. 
+2. `MapboxNavigation.xcodeproj` 
 
 ### Using Carthage
```

Comment:
This section needs a reminder to add any new file to the MapboxNavigation or MapboxCoreNavigation target in MapboxNavigation.xcodeproj after adding it to the file system. Otherwise, the two build systems can get out of sync. (That was the main benefit to keeping the two build systems in the same project originally, but at this point the downsides outweigh that benefit.)

Reply:
Updated, please check.